src/early_multicellular_stage/Microbe.Multicellular.cs

```diff
@@ -131,6 +131,23 @@ public void BecomeFullyGrownMulticellularColony()
         }
     }
 
+    /// <summary>
+    ///   Perform an action for all members of this cell's colony other than this cell.
+    /// </summary>
+    public void PerformForAllColonyMembers(Action<Microbe> action)
```

Comment:
```suggestion
    public void PerformForOtherColonyMembers(Action<Microbe> action)
```

The name having "all" in it, but the comment then clarifying it doesn't actually mean "all" is pretty confusing so I'd recommend this rename.

Reply:
I fixed this myself and went with an even more verbose naming.